packages/module/src/LongTextTooltip/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default } from './LongTextTooltip';
+export { default as LongTextTooltip } from './LongTextTooltip';
```

Comment:
we should not provide a named export for the component, just a default one

Reply:
all set.